tools/javadoc2mdoc/javadoc2mdoc.csproj

```diff
@@ -5,6 +5,7 @@
     <OutputType>Exe</OutputType>
     <RootNamespace>Xamarin.Android.Tools.JavadocToMDoc</RootNamespace>
     <AssemblyName>javadoc-to-mdoc</AssemblyName>
+    <IntermediateOutputPath>$(BaseIntermediateOutputPath)$(Configuration)\$(TargetFramework)\</IntermediateOutputPath>
```

Comment:
For this one could you remove this instead?
https://github.com/xamarin/xamarin-android/blob/d73db9d8b40a2530b04c0c1d5ecf02ce5f51aa44/tools/javadoc2mdoc/javadoc2mdoc.csproj#L4

Reply:
I don't know, I think it would append it to our `<OutputPath>` which would break stuff?

```
<OutputPath>$(XAInstallPrefix)xbuild\Xamarin\Android\</OutputPath>
```

Reply:
I just removed the net6.0 build from this project as it will remain a legacy-only feature, so we may not need any change here.

Reply:
Good point, this has changed since I wrote this.  😁 

https://github.com/xamarin/xamarin-android/pull/7062